package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@canonical/react-components",
-  "version": "0.37.2",
+  "version": "0.38.0",
```

Comment:
In React components we usually don't bump the version in each PR, because there always needs to be a "Prepare release" PR to update storyboook docs, so we do it at that point.

Reply:
Done :+1: